### ISSUE TITLE
Attachment // files content_type and file name empty

### ISSUE THREAD

**Issue by steffenmllr** [nylas/sync-engine#185]
Hey there,

trying to get the filename and the content_type for an attachment within a message.
It's a google account and this happens in the hosted version and within vagrant.

```json
        "files": [
            {
                "content_type": "application/octet-stream",
                "filename": null,
                "id": "f49myfuq14vymt5qsxciao9on",
                "size": 8120
            }
        ],
```

When I look the message up in gmail it's a pdf and it has a file name

**Comment by grinich** [comment]
Hm, that certainly sounds like a bug! I'd guess this is an issue with our MIME parser somehow failing on a header. 

It would be super helpful if you could attach the [full message with headers](https://support.google.com/groups/answer/75960?hl=en) to this thread, as long as it's not too personal. Thanks @steffenmllr !

**Comment by steffenmllr** [comment]
It's not really private, it was just a test mail with a test attachment:
https://gist.github.com/steffenmllr/5840ca0797a7aa4d5728

**Comment by grinich** [comment]
Got it, thanks. This is a known bug in the underlying [flanker](https://github.com/mailgun/flanker) MIME parsing library. We'll put together a fix. Thanks for the great test case! :)

**Comment by khamidou** [comment]
Fixed in 32ea5a9. Thanks again @steffenmllr for the detailed bug report!
